apps/web/app/_utils.tsx

```diff
@@ -143,3 +143,26 @@ export const generateAppMetadata = async (
     },
   };
 };
+
+export function checkIfBookingPage(): boolean {
+  const headersList = headers();
+  const matchedPath = headersList.get("x-matched-path"); // Exists only in Staging and Prod
```

Comment:
If it exists only in Staging and Prod, what happens in dev?

Reply:
in dev, `checkIfBookingPage()` will always be false